chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -599,7 +600,10 @@ impl PeerManagerActor {
             last_time_received_message: cp.last_time_received_message.load(),
             connection_established_time: cp.established_time,
             peer_type: cp.peer_type,
-            nonce: cp.edge.load().nonce(),
+            nonce: match graph.local_edges.get(&cp.peer_info.id) {
+                Some(e) => e.nonce(),
+                None => 0,
```

Comment:
when does this case happen ?

Reply:
when an edge is removed from the graph (when it expires, for example).